v2/cmd/asoctl/pkg/importresources/importable_resource.go

```diff
@@ -36,18 +36,10 @@ type ImportableResource interface {
 	// ctx allows for cancellation of the import.
 	// log allows information about progress to be reported
 	Import(
-		ctx context.Context,
-		log logr.Logger,
-	) (ImportedResource, error)
-
-	// FindChildren returns any child resources that need to be imported.
-	// ctx allows for cancellation of the import.
-	// Returns any additional resources that also need to be imported, as well as any errors that occur.
-	// Partial success is allowed, but the caller should be notified of any errors.
-	FindChildren(
```

Comment:
Having `FindChildren()` as a separate method introduced some temporal coupling in the interface, as you had to remember to call `Import()` first. I've collapsed both into a single call.